code/modules/research/designs/design_rig_module.dm

```diff
@@ -4,7 +4,6 @@
 	category = "Modules"
 	req_tech = list(TECH_MAGNET = 2, TECH_ENGINEERING = 5)
 	materials = list(MATERIAL_STEEL = 4000, "plastic" = 2500, MATERIAL_GLASS = 2000, MATERIAL_GOLD = 1000)
-	build_type = STORE
```

Comment:
RIG modules are intended to all be in the store, not just a select few of them

if there are specific ones that need to be excluded it can be removed from them